frontend/src/modules/WellLogViewer/utils/trajectory.ts

```diff
@@ -0,0 +1,22 @@
+import { IntersectionReferenceSystem } from "@equinor/esv-intersection";
+import _ from "lodash";
```

Comment:
Replace import _ from "lodash"; with import of correct utils, as done for rest of the module code in WellLogViewer